packages/plugins/rubygem-foreman_ansible/rubygem-foreman_ansible.spec

```diff
@@ -46,6 +44,8 @@ Provides: foreman-plugin-%{plugin_name} = %{version}
 # start package.json devDependencies BuildRequires
 BuildRequires: %{?scl_prefix}npm(@babel/core) >= 7.7.0
 BuildRequires: %{?scl_prefix}npm(@babel/core) < 8.0.0
+BuildRequires: %{?scl_prefix}npm(@testing-library/user-event) >= 13.2.1
+BuildRequires: %{?scl_prefix}npm(@testing-library/user-event) < 14.0.0
```

Comment:
```
DEBUG util.py:600:  No matching package to install: 'npm(@testing-library/user-event) >= 13.2.1'
DEBUG util.py:600:  No matching package to install: 'npm(@testing-library/user-event) < 14.0.0'
```

is this something we actually need? or is this for testing only?

aka: should we add `@testing-library` to excluded prefixes in https://github.com/theforeman/foreman/blob/develop/package-exclude.json?

Reply:
looking at https://testing-library.com/, excluding all these packages seems sane?

Reply:
yeah, I think it make sense :+1: Will do